README.md

```diff
@@ -370,7 +379,7 @@ cd ../../../infra/terraform/platform
 
   ```
   cd ../../../infra/kops/clusters/$ENVNAME
-  kops create -f cluster.yml
+  kops create -f cluster.$ENV.rendered.yml
```

Comment:
We should probably be consistent on using $ENV or $ENVNAME (compare with previous line). I'm fine with $ENV, if you want to change the other occurrences?